pkg/web/apps/apps_handler.go

```diff
@@ -0,0 +1,34 @@
+package apps
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo"
+)
+
+type (
+	// Handler instance
+	Handler struct {
```

Comment:
https://blog.golang.org/package-names

> Avoid stutter. Since client code uses the package name as a prefix when referring to the package contents, the names for those contents need not repeat the package name. The HTTP server provided by the http package is called Server, not HTTPServer. Client code refers to this type as http.Server, so there is no ambiguity.

Reply:
But the link I posted still stands. Because your suggestion would mean the struct would be called like this:

```go
x := apps.AppsHandler ...
```

Which is stuttering. Anyway, the name has had to change with the new changes we made, it is now `HTTPHandler` so it is called like:

```go
x := apps.HTTPHandler ...
```